card.rb

```diff
@@ -1,19 +1,40 @@
 class Card
 
-  def card(args = {})
+  attr_reader :term, :def
+  attr_accessor :complete
+
+  def initialize(args = {})
     @term = args[:term]
     @def = args[:def]
-    @complete = fetch(:complete, false)
+    @complete = args.fetch(:complete, false)
   end
 
   def mark_complete
+    @complete = true
+  end
 
+  def to_s
+    "#{self.term}: #{self.def}"
   end
 
-  def to.s
+  def guess(string)
+    mark_complete if self.term == string
+    complete
+  end
 
+  def got_correct?
```

Comment:
@AgentWang007 Since we have complete as an accessible variable, we probably don't need this; we can just refer to that variable.